packages/inter-protocol/src/auction/auctioneer.js

```diff
@@ -26,6 +26,7 @@ import { makeAuctionBook } from './auctionBook.js';
 import { AuctionState } from './util.js';
 import { makeScheduler } from './scheduler.js';
 import { auctioneerParamTypes } from './params.js';
+import { appendToStoredArray } from '@agoric/store/src/stores/store-utils.js';
```

Comment:
How about we make it a direct export from `@agoric/store` and avoid the deep import?

Reply:
Probably better but not worth another CI spin imo. I expect we'll clean up deep imports after or as part of refactoring the repo.